src/installer.ts

```diff
@@ -159,6 +161,8 @@ async function downloadBuf(version: string): Promise<string> {
   }
 }
 
+// semverCoerce coerces the version to a semver version. This is useful for
+// custom versions that are not semver compliant e.g. "v1.32.0-beta.1".
```

Comment:
`v1.32.0-beta.1` should be a valid semver, is it not?

https://semver.org/spec/v2.0.0.html#spec-item-9

Reply:
It is, I thought the lib didn't handle them but it does too: https://www.npmjs.com/package/semver. I'll remove this coerce as it was only needed for the `-beta` version and fix this documentation.